src/main/java/dwbh/api/repositories/GroupRepository.java

```diff
@@ -115,6 +115,46 @@ public Group createGroup(
         .build();
   }
 
+  /**
+   * Updates a group
+   *
+   * @param groupId group's id
+   * @param name group's name
+   * @param visibleMemberList indicates if the group allows the members to see the member list
+   * @param anonymousVote indicates if the group allows anonymous votes
+   * @param daysOfWeek days of the week when reminders are sent
+   * @param time moment of the day when reminders are sent
+   * @return The updated {@link Group}
+   * @since 0.1.0
+   */
+  public Group updateGroup(
```

Comment:
This should be solved by an `UPSERT` operation. I think in Jooq is `mergeInto(...)` method. This way there should one persistence method for insert/update

Reply:
Good idea! Changed

Btw, Postgres doesn't support mergeInto. It is with insertInto / onConflict.